app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Alignment, Direction, Stack } from '@kibalabs/ui-react';
+
+
+export const TokenPage = (): React.ReactElement => {
+//   const routeParams = useRouteParams();
+//   const address = routeParams.address as string;
+//   const tokenId = routeParams.address as string;
+
+  return (
+    <Stack direction={Direction.Vertical} isFullWidth={true} isFullHeight={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Start} isScrollableVertically={true} />
```

Comment:
would be good to add a Text in here with something like "Token page for {collectionAddress}/{tokenId}